### ISSUE TITLE
[REQUEST]: Compare devices or device panes in diff view

### ISSUE THREAD

**Issue by pauleonix** [compiler-explorer/compiler-explorer#4359]
### Describe the bug

The device pane has recently been added for "CUDA C++" mode. It makes it easier to understand what assembly one is seeing, but as long as there is no way of choosing these device panes in the Diff view this is a functional regression as one cannot compare the PTX or SASS in the DIff View anymore.

### Steps to reproduce

1. Visit https://cuda.godbolt.org/
2. Click `Add...` -> `Diff View`
3.  Try to get the new diff pane to display the code from the device pane.

### Expected behavior

While one could add the device panes to the first drop-down menu (showing the compiler) of the diff pane, it might be better to have a "PTX" and a "SASS" option in the second drop-down menu (showing "Assembly" per default).

### Reproduction link

https://cuda.godbolt.org/z/MG7e153Wv

### Screenshots

![Screenshot from 2022-11-28 20-53-13](https://user-images.githubusercontent.com/23128868/204368381-a95e6321-f966-4f8c-a4ca-1cc3d7d2b6ee.png)

**Comment by jrhemstad** [comment]
Agreed. I would also like to ability to do diffs of the device view panel.

**Comment by partouf** [comment]
We might be able to offer comparison without the physical device pane as it's information that always comes back with the compilation no matter if the pane is there or not

**Comment by partouf** [comment]
This has been implemented on April 19, 2023 via #4973

**Comment by pauleonix** [comment]
Oh yeah, sorry, forgot that this issue existed!
